lang/de_DE.trn

```diff
@@ -2044,10 +2044,10 @@ You may not be able to access all functions at this resolution.
 t950 MCEdit unterstützt keine Fensterauflösungen unter 1000x700.
 Du wirst unter dieser Auflösung vielleicht nicht auf alle Funktionen zugreifen können.
 o951 Found a previous portable installation
-t951 
+t951 Eine vorherige portabel Installation wurde gefunden
```

Comment:
`portable` is english, `portabel` is (old) german. We already use `portabel`, so for consistency, i also used that.